### PR TITLE
fix(support): minor ux improvements

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html [hunk 1]
```diff
@@ -80,17 +80,25 @@
...
-      <oui-button data-variant="secondary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_no' | translate }}"
-                  data-on-click="$ctrl.submitForm(false)">
-      </oui-button>
-      <oui-button data-variant="primary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_yes' | translate }}"
-                  data-on-click="$ctrl.submitForm(true)">
-      </oui-button>
+      <div data-ng-if="$ctrl.guides.length || $ctrl.tips.length">
+        <p class="mt-4"
```
Comment (antleblanc) [inline packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html]: Indentation size isn't correct. Should be 4 spaces for `*.html` files.

ref: https://github.com/ovh-ux/manager/blob/master/.editorconfig#L22

File: packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html [hunk 2]
```diff
@@ -45,7 +45,7 @@
       </oui-select>
       <oui-checkbox data-model="$ctrl.isUnknownService"
                     data-on-change="$ctrl.onUnknownServiceChange()">
-        {{ 'ovhManagerSupport_new_service_unknown' | translate }}
+        {{:: 'ovhManagerSupport_new_service_unknown' | translate }}
```
Comment (FredericEspiau) [inline packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html]:
```suggestion
        <span data-translate="ovhManagerSupport_new_service_unknown"></span>
```

File: packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html [hunk 3]
```diff
@@ -80,17 +80,25 @@
...
-      <oui-button data-variant="primary"
-                  data-text="{{ 'ovhManagerSupport_new_helpfull_yes' | translate }}"
-                  data-on-click="$ctrl.submitForm(true)">
-      </oui-button>
+      <div data-ng-if="$ctrl.guides.length || $ctrl.tips.length">
+        <p class="mt-4"
+           data-translate="ovhManagerSupport_new_helpfull">
+        </p>
+        <oui-button data-variant="secondary"
+                    data-text="{{:: 'ovhManagerSupport_new_helpfull_no' | translate }}"
```
Comment (FredericEspiau) [inline packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html]: Don't use `text`, use transpilation instead:
````js
<oui-button
  data-variant="secondary"
  data-on-click="$ctrl.submitForm(false)">
  <span data-translate="ovhManagerSupport_new_helpfull_no"></span>
</oui-button>
````

Also, looks like this submits a form, then the button `type` should be `submit`;)

File: packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html (hunk 1 above)
Comment (frenautvh) [reply packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html]: i indented all the templates in new-ticket parts with 2 spaces ; i'll do another PR to fix all the indentation

**Pull request merged**